README.md

```diff
@@ -73,23 +70,9 @@ After you're finished please remove all the comments and instructions!
 > Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
[OPTIONAL]

Good job so far 👍🏽. I think it will be better if you remove all the placeholder texts in each section of your readme. These texts are to guide you on what to add to each section, so therefore, you should remove them to make your readme professional.